lib/animina/accounts/resources/basic_user.ex

```diff
@@ -33,6 +33,7 @@ defmodule Animina.Accounts.BasicUser do
     attribute :gender, :string, allow_nil?: false
     attribute :height, :integer, allow_nil?: false
     attribute :mobile_phone, :string, allow_nil?: false
+    attribute :language, :string, allow_nil?: false
```

Comment:
The `User` resource needs the `:language` attribute too.

Reply:
I've added the language attribute to the User resource